cmd/di.go

```diff
@@ -525,8 +523,13 @@ func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation
 }
 
 func (di *Dependencies) bootstrapMetrics(options node.Options) {
+	loader := &upnp.GatewayLoader{}
+
+	// warm up the loader as the load takes up to a couple of secs
```

Comment:
What will be if we will not warm up it? Since we are starting it in a goroutine app can reach the place where it's needed faster than a couple of seconds.

Reply:
It doesnt matter in that case, the once will be locked there and we'll wait if needed then... But in general this is a "good enough" solution